server/website/website/tasks/async_tasks.py

```diff
@@ -180,13 +178,6 @@ def calc_next_knob_range(algorithm, knob_info, newest_result, good_val, bad_val,
         else:
```

Comment:
When the config is valid why do we use the expected_value instead of the last_value?